frontend/src/lib/k8s/apiProxy.ts

```diff
@@ -607,23 +616,30 @@ function singleApiFactory(group: string, version: string, resource: string) {
   const apiRoot = getApiRoot(group, version);
   const url = `${apiRoot}/${resource}`;
   return {
-    list: (cb: StreamResultsCb, errCb: StreamErrCb, queryParams?: QueryParameters) => {
+    list: (
+      cb: StreamResultsCb,
+      errCb: StreamErrCb,
+      queryParams?: QueryParameters,
+      cluster?: string
+    ) => {
       if (isDebugVerbose('k8s/apiProxy@singleApiFactory list')) {
-        console.debug('k8s/apiProxy@singleApiFactory list', { queryParams });
+        console.debug('k8s/apiProxy@singleApiFactory list', { cluster, queryParams });
       }
 
-      return streamResults(url, cb, errCb, queryParams);
+      return streamResultsForCluster(url, { cb, errCb, cluster }, queryParams);
     },
     get: (name: string, cb: StreamResultsCb, errCb: StreamErrCb, queryParams?: QueryParameters) =>
```

Comment:
I wonder why the GET don't get a cluster?

Reply:
The main motivation was the listing functions. I think it makes sense to have the `get` and other functions receive the cluster too.
Let's do that in a later iteration.

Reply:
ok